tetris.py

```diff
@@ -125,38 +124,47 @@ def new_block(self):
             self.current_block_shape = self.upcoming_block_shape
         self.current_block_center = (int(game_width / 2), -2)
         self.upcoming_block_shape = random.choice(shape_names)
+        self.index = 0
 
-    def get_current_block(self):
+    def get_current_block(self, index):
```

Comment:
```suggestion
    def get_current_block(self):
```

The `index` argument is never actually used. The method uses `self.index` which comes from `self` argument , not from `index` argument. This also simplifies all code that calls this method.